site/en/guide/distributed_training.ipynb

```diff
@@ -503,14 +503,14 @@
       "source": [
         "## Use tf.distribute.Strategy with Keras Model.fit\n",
         "\n",
-        "`tf.distribute.Strategy` is integrated into `tf.keras`, which is TensorFlow's implementation of the [Keras API specification](https://keras.io). `tf.keras` is a high-level API to build and train models. By integrating into the `tf.keras` backend, it's seamless for you to distribute your training written in the Keras training framework [using Model.fit](/keras/customizing_what_happens_in_fit.ipynb).\n",
+        "`tf.distribute.Strategy` is integrated into `tf.keras`, which is TensorFlow's implementation of the [Keras API specification](https://keras.io/api/). `tf.keras` is a high-level API to build and train models. By integrating into the `tf.keras` backend, it's seamless for you to distribute your training written in the Keras training framework [using Model.fit](keras-io/guides/ipynb/customizing_what_happens_in_fit.ipynb).\n",
         "\n",
         "Here's what you need to change in your code:\n",
         "\n",
         "1. Create an instance of the appropriate `tf.distribute.Strategy`.\n",
         "2. Move the creation of Keras model, optimizer and metrics inside `strategy.scope`.\n",
         "\n",
-        "TensorFlow distribution strategies support all types of Keras models—[Sequential](/keras/sequential_model.ipynb), [Functional](/keras/functional.ipynb), and [subclassed](/keras/custom_layers_and_models.ipynb).\n",
+        "TensorFlow distribution strategies support all types of Keras models—[Sequential](/keras-io/guides/ipynb/sequential_model.ipynb), [Functional](keras-io/guides/ipynb/functional_api.ipynb), and [subclassed](keras-io/guides/ipynb/making_new_layers_and_models_via_subclassing.ipynb).\n",
```

Comment:
Relative linking (https://www.tensorflow.org/community/contribute/docs_style#links_in_markdown_and_notebooks)

Let's try the following:

```suggestion
        "TensorFlow distribution strategies support all types of Keras models—[Sequential](./keras/sequential_model.ipynb), [Functional](./keras/functional.ipynb), and [subclassed](./keras/custom_layers_and_models.ipynb)\n",
```